recipes-extended/xen/files/oxenstored.conf

```diff
@@ -0,0 +1,45 @@
+# OpenXT xenstored config
+
+# Where the pid file is stored
+pid-file = /var/run/xenstored.pid
+
+# Randomly failed a transaction with EAGAIN. Used for testing Xs user
+test-eagain = false
```

Comment:
This is the only param that is different. @jean-edouard and I are looking what this does and it seems like the logic in xenstored is backwards/broken. We need to look at this more closely but right now it looks like setting this to true is really disabling it.